src/js/injection_script.js

```diff
@@ -2706,7 +2706,7 @@ domainBypass('apkadmin.com', () => {
 
 	domainBypass("tei.ai", () => {
 		const token = document.querySelector('#link-view [name="token"]').value;
-		const decoded = atob(token.substring(token.indexOf("aH")));
+		const decoded = atob(token.substring(token.indexOf("aHR0")));
 		const page = decoded.split('http').pop();
```

Comment:
this needs to be done better, for me when i use example link: https://tei.ai/HMguaHDSHqC the atob on the part behind aHR0 returns: 'ps://share.vidyard.com/watch/y6sdUGsVX92vYtJ1PEf6TP?autoplay=1', making the split fail, we may have to fully rework on this

Reply:
hmm why is the aH not working seperately?